docker-compose-gateway-aws.yaml

```diff
@@ -9,6 +9,7 @@ services:
     environment:
       MINIO_ROOT_USER: "12345678"
       MINIO_ROOT_PASSWORD: "12345678"
+      # MINIO_API_SELECT_PARQUET: "on"
```

Comment:
this should be changed to:

```
MINIO_API_SELECT_PARQUET: "${MINIO_API_SELECT_PARQUET}"
```

then to enable parquet you will need to set the following in `.env` file:

```
MINIO_API_SELECT_PARQUET=on
```